bucket/rustup-msvc.json

```
@@ -18,7 +18,8 @@
         "script": [
             "[Environment]::SetEnvironmentVariable('CARGO_HOME', \"$persist_dir\\.cargo\", 'Process')",
             "[Environment]::SetEnvironmentVariable('RUSTUP_HOME', \"$persist_dir\\.rustup\", 'Process')",
-            "Invoke-ExternalCommand -Path \"$dir\\rustup-init.exe\" -Args '-y --no-modify-path' | Out-Null"
+            "$arch=$(If ($architecture -eq '64bit') {'x86_64'} Else {'i686'})",
+            "Invoke-ExternalCommand -Path \"$dir\\rustup-init.exe\" -Args \"-y --no-modify-path --default-host $arch-pc-windows-msvc\" | Out-Null"
```

```suggestion
            "$host_arch = if ($architecture -eq '64bit') {'x86_64'} else {'i686'}",
            "Invoke-ExternalCommand -Path \"$dir\\rustup-init.exe\" -Args \"-y --no-modify-path --default-host $host_arch-pc-windows-msvc\" | Out-Null"
```

$arch is used internally, so use $host_arch. Also, parenthesis is not required.

Same below.